docs/intro/tutorial01.txt

```diff
@@ -84,9 +84,9 @@ Let's look at what :djadmin:`startproject` created:
         manage.py
         mysite/
             __init__.py
+            asgi.py
```

Comment:
Now the file descriptions in the `These files are:` are not in the same order. I wanted to add `asgi.py` next to the `wsgi.py` to make it easier to read for newcomers.

Reply:
I agree that this change feels unnecessary, and that if we do merge it for whatever reason, the following section needs to be adjusted to use the same order as the filesystem list:
![image](https://github.com/django/django/assets/124304/326ab6ab-d899-42ec-8396-ea1603f32382)

Reply:
Okay, fair enough, I will revert this bit.